### ISSUE TITLE
WCAG 4.1.2 Name, Role, Value

### ISSUE THREAD

**Issue by ToneMereteBruvik** [ufal/dspace-angular#35]
Lighthouse poitet out these error on the home page:

Each ARIA role supports a specific subset of aria-* attributes. Mismatching these invalidates the aria-attributes: ngb-carousel.carousel.slide:

![Image](https://github.com/user-attachments/assets/fd2e095a-3c5d-479b-b15c-8d1ad41071d8)

[aria-hidden="true"] elements contain focusable descendents

![Image](https://github.com/user-attachments/assets/1f768891-8687-4ec4-aebb-059a90012150)

When I testet the "Edit Submisson" page (https://lindat.mff.cuni.cz/repository-ng/workspaceitems/5604/edit), there were 7 failing elements, starting with:

![Image](https://github.com/user-attachments/assets/96769e8d-2c4d-4c0a-a35f-1ce587a37925)
